action.yml

```diff
@@ -15,6 +15,9 @@ inputs:
     description: 'If true, will throw an error (and crash CI) when there is an error about aliases link'
     required: false
     default: "false"
+  extraAliases:
+    description: 'List of alias which can be created dynamically'
```

Comment:
List of additional aliases to create, will be added to the aliases specified in the `vercel.json:aliases` file